experiments/long_runs/snowy_land.jl

```diff
@@ -206,8 +206,8 @@ function setup_model(FT, start_date, stop_date, Δt, domain, earth_param_set)
     else
         modis_lai_ncdata_path = ClimaLand.Artifacts.modis_lai_multiyear_paths(;
             context = nothing,
-            start_date = start_date + Second(t0),
-            end_date = start_date + Second(t0) + Second(tf),
+            start_date,
+            end_date = stop_date,
```

Comment:
this was a bug introduced in an earlier PR but not caught since we dont run long runs each step